docs/user_guide/capsules/index.md

```diff
@@ -41,6 +41,12 @@ In the simplest use case, a user may only want to use the Default Capsule. Large
 
 See [Capsule Installation](/docs/{{ site.version }}/installation/capsule.html)
 
+## Removal
+
+To stop all services and remove all Katello and Foreman related packages, run the following command as root on the capsule:
+
+  * `capsule-remove`
```

Comment:
We could drop the '*' here as that makes it appear as a list and just do a code-block:

```
capsule-remove
````